.circleci/config.yml

```diff
@@ -170,15 +216,30 @@ jobs:
     steps:
       - deploy:
           hostname: "ssh.prod.avniproject.org"
+          
+tags_filter: &tags_filter
+  filters:
+    tags:
+      only: /.*/
 
 workflows:
   version: 2
   test_and_build_and_deploy:
     jobs:
-    - test
+    - test:
+        <<: *tags_filter
     - build:
+        <<: *tags_filter
+        requires:
+          - test     
+    - build_docker_image:
```

Comment:
@yenugukeerthana Since the secrets are added under context, Please add the context entry here. Ref: https://circleci.com/docs/contexts/

Reply:
Done @mohan-13